src/games/erdos_renyi.c

```diff
@@ -42,6 +42,68 @@
  * fixed degrees, etc.)</para>
  */
 
+/* This implementation is used only with very large vertex counts, above
+ * sqrt(MAX_EXACT_REAL) ~ 100 million, when the default implementation would
+ * fail due to overflow. While this version avoids overflow and uses less memory,
+ * it is also slower than the default implementation. */
+static igraph_error_t gnp_large(
+    igraph_t *graph, igraph_integer_t n, igraph_real_t p,
+    igraph_bool_t directed, igraph_bool_t loops, igraph_integer_t ecount_estimate
+) {
+
+    igraph_vector_int_t edges;
+    int iter = 0;
+
+    printf("Yay!\n");
```

Comment:
This line should be removed :)

Reply:
🤦 There's a reason I'm asking for reviews!